.github/workflows/test.yml

```diff
@@ -57,6 +57,15 @@ jobs:
       - name: Test
         run: cargo nextest run --workspace --nocapture && cargo test --doc --workspace
 
+      - name: Cycle tracker report
+        continue-on-error: true
+        env:
+          # Make sure we get useful output and a handy summary at the end
+          RUST_LOG: warn,cycle_tracker=info
+          NEXTEST_SUCCESS_OUTPUT: immediate-final
+          NEXTEST_FAILURE_OUTPUT: immediate-final
+        run: cargo nextest run -p pessimistic-proof-client --test cycle-tracker
```

Comment:
Could you make this test under a `nextest` test-group?
in order to not have it executed when running nextest globally

Reply:
> Could you make this test under a `nextest` test-group? in order to not have it executed when running nextest globally

It seems to me that nextest groups are there mainly to limit the number of resources (CPU cores) used by a particular set of tests. Not sure how it applies here. I instead applied the `#[ignore]` attribute to prevent it from being run by default.